### PR TITLE
Add small doc for splitter

### PR DESCRIPTION
**Pull request by v-ahuja**
It was difficult to use `splitter` (especially with a `dataframe`) without seeing the usage in the course textbook.

I've added a small sentence here that will at least provide some intuition for how it should be used. Ideally, I'd like more documentation including how splitter processes a `dataframe` or some other data source, but this is the start that i could think of. suggestions welcome.

Also:

Ideally, instead of showing up as `No content` in the table, we could have the docs for `splitter`, `get_x` and `get_y` show up along with the rest of the args. However, this is more involved and looks like it needs changes to `nbdev`. I suggest this as a follow up enhancement to `nbdev`, which I can take a crack at.

### CODE REVIEW

**Comment by v-ahuja** [comment]
@jph00 thanks in advance for reviewing.

File: fastai/data/block.py [hunk 1]
```diff
@@ -77,7 +77,9 @@ def _zip(x): return L(x).zip()
 @docs
 @funcs_kwargs
 class DataBlock():
-    "Generic container to quickly build `Datasets` and `DataLoaders`"
```
Comment (jph00) [inline fastai/data/block.py:80]: Please don't change the formatting of existing fastai code, and when you add code please try to follow fastai standards. Our standard for docstrings is to use a single line and a single pair of double quotes.

**Review by jph00 — CHANGES_REQUESTED** [review]
Thanks for your contribution. Just one requested change please.

**Comment by v-ahuja** [comment]
changed. tks.

**Comment by jph00** [comment]
Thanks!

**Pull request merged**